src/index.ts

```diff
@@ -341,25 +351,42 @@ class Spanner extends GrpcService {
     this.getInstancesStream = paginator.streamify('getInstances');
   }
 
+  createInstance(
+    name: string,
+    config: CreateInstanceRequest
+  ): Promise<CreateInstanceResponse>;
+  createInstance(
+    name: string,
+    config: CreateInstanceRequest,
+    callback: CreateInstanceCallback
+  ): void;
   /**
    * Config for the new instance.
    *
    * @typedef {object} CreateInstanceRequest
    * @property {string} config The name of the instance's configuration.
-   * @property {number} nodes The number of nodes allocated to this instance.
+   * @property {number} [nodes] The number of nodes allocated to this instance.
+   *     Defaults to 1.
+   * @property {Object.<string, string>} [labels] Labels are a flexible and
+   *     lightweight mechanism for organizing cloud resources into groups that
+   *     reflect a customer's organizational needs and deployment strategies.
+   *     Cloud Labels can be used to filter collections of resources. They can
+   *     be used to control how resource metrics are aggregated. And they can
+   *     be used as arguments to policy management rules (e.g. route,
+   *     firewall, load balancing, etc.).
    */
   /**
    * @typedef {array} CreateInstanceResponse
    * @property {Instance} 0 The new {@link Instance}.
-   * @property {Operation} 1 An {@link Operation} object that can be used to check
+   * @property {Operation} 1 An operation object that can be used to check
```

Comment:
Why are the links removed here and below?

Reply:
`Operation` is not defined, link it does not provide anything during docs generation.